layers/vk_mem_alloc.h

```diff
@@ -8066,12 +8066,20 @@ VkResult VmaBlockMetadata_Generic::CheckCorruption(const void* pBlockData)
             if(!VmaValidateMagicValue(pBlockData, it->offset - VMA_DEBUG_MARGIN))
             {
                 VMA_ASSERT(0 && "MEMORY CORRUPTION DETECTED BEFORE VALIDATED ALLOCATION!");
+#if defined(VK_EXT_debug_report)
                 return VK_ERROR_VALIDATION_FAILED_EXT;
+#else
+                return VK_ERROR_VALIDATION_FAILED;
```

Comment:
If we added VK_ERROR_VALIDATION_FAILED_EXT to vksc_compatibility.h we could get away without needing most of the edits in this file.

Reply:
Good idea, I'll update.